.github/CODEOWNERS

```diff
@@ -16,85 +16,73 @@
 *   @open-telemetry/specs-semconv-maintainers @open-telemetry/specs-semconv-approvers
 
 # Schemas and schema file tooling
-/schemas/                       @open-telemetry/specs-semconv-approvers @tigrannajaryan
-/internal/tools/schema_check.sh @open-telemetry/specs-semconv-approvers @tigrannajaryan
+/schemas/                                 @open-telemetry/specs-semconv-approvers @tigrannajaryan
```

Comment:
For later:  We should think about using the tooling SIG or a separate set of folks to own the schema definitions.

cc @lquerel @tigrannajaryan @open-telemetry/specs-semconv-maintainers

Reply:
created https://github.com/open-telemetry/semantic-conventions/issues/1410 to track it and some other areas we forgot to mention